### PR TITLE
Enahncement/inversion

### PR DESCRIPTION
**Pull request by ric-jon**
Small improvement on how inversions look.

### CODE REVIEW

File: src/main/java/application/controllers/GraphController.java [hunk 1]
```diff
@@ -145,6 +147,23 @@ public void update(Object ref, int depth) {
...
+                        - e.getLine().startXProperty().get();
+                double yLength = e.getLine().endYProperty().get()
+                        - e.getLine().startYProperty().get();
+                double length = Math.sqrt(Math.pow(xLength, 2) + Math.pow(yLength, 2));
+                System.out.println(e.getSource().toString() + " " + length);
+                if (length > MAX_EDGE_LENGTH) {
+                    e.getLine().getStrokeDashArray().addAll(3d, 17d);
+                    e.getLine().setOpacity(0.2d);
+                    double newY = e.getSource().getLayoutY()
+                            + (e.getSource().getLayoutY() - (screenSize.getHeight() - 150) / 2) * 2.5;
```
Comment (ghost) [inline src/main/java/application/controllers/GraphController.java]: From CheckStyle: Line is longer than 100 characters (found 102).

File: src/main/java/application/controllers/GraphController.java [hunk 2]
```diff
@@ -81,8 +83,8 @@ public GraphController(MainController m) {
     public void focus(Cell prevClick) {
         prevClick.resetFocus();
         for (Cell c : graph.getModel().getAllCells()) {
-            if ((c.getCellId() == prevClick.getCellId()) ||
-                    (c.getCellId() > prevClick.getCellId())) {
+            if ((c.getCellId() == prevClick.getCellId())
```
Comment (ghost) [inline src/main/java/application/controllers/GraphController.java:86]: From PMD: useless parentheses

File: src/main/java/application/controllers/GraphController.java [hunk 3]
```diff
@@ -81,8 +83,8 @@ public GraphController(MainController m) {
...
+                    || (c.getCellId() > prevClick.getCellId())) {
```
Comment (ghost) [inline src/main/java/application/controllers/GraphController.java:87]: From PMD: useless parentheses

**Comment by Warnars** [comment]
kek G, merge incoming

**Pull request merged**